usaspending_api/etl/management/commands/load_query_to_delta.py

```diff
@@ -85,6 +89,23 @@
         "custom_schema": "recipient_hash STRING",
         "column_names": [x for x in list(RECIPIENT_PROFILE_DELTA_COLUMNS) if x != "id"],
     },
+    "int.sam_recipient": {
+        "model": None,
+        "is_from_broker": True,
+        "source_query": sam_recipient_load_sql_string,
+        "source_database": None,
+        "source_table": None,
+        "destination_database": "int",
```

Comment:
Shouldn't this be going into `raw`?

Reply:
changed it to raw; if needs to be changed to int, can be done.